examples/shapes.rs

```diff
@@ -7,34 +7,40 @@ use three::Object;
 
 fn main() {
     let mut win = three::Window::new("Three-rs shapes example");
-    let mut cam = win.factory.perspective_camera(75.0, 1.0 .. 50.0);
+    let cam = win.factory.perspective_camera(75.0, 1.0 .. 50.0);
     cam.set_position([0.0, 0.0, 10.0]);
 
-    let mut mbox = {
+    let mbox = {
         let geometry = three::Geometry::cuboid(3.0, 2.0, 1.0);
         let material = three::material::Wireframe { color: 0x00FF00 };
         win.factory.mesh(geometry, material)
     };
     mbox.set_position([-3.0, -3.0, 0.0]);
-    mbox.set_parent(&win.scene);
+    win.scene.add(&mbox);
 
-    let mut mcyl = {
+    let mcyl = {
         let geometry = three::Geometry::cylinder(1.0, 2.0, 2.0, 5);
         let material = three::material::Wireframe { color: 0xFF0000 };
         win.factory.mesh(geometry, material)
     };
     mcyl.set_position([3.0, -3.0, 0.0]);
-    mcyl.set_parent(&win.scene);
+    win.scene.add(&mcyl);
 
-    let mut msphere = {
+    let msphere = {
         let geometry = three::Geometry::uv_sphere(2.0, 5, 5);
         let material = three::material::Wireframe { color: 0xFF0000 };
         win.factory.mesh(geometry, material)
     };
     msphere.set_position([-3.0, 3.0, 0.0]);
-    msphere.set_parent(&win.scene);
+    win.scene.add(&msphere);
 
-    let mut mline = {
+    // test removal from scene
+    win.scene.remove(&mcyl);
```

Comment:
Is it supposed to be here?

Reply:
yes - it's a small test for object removal. May not be the best place, but better than nothing